.github/workflows/check.yaml

```diff
@@ -68,3 +68,8 @@ jobs:
         uses: ethersphere/beeload-action@v1
         with:
           preview: 'true'
+
+      - name: Upload to testnet
+        uses: ethersphere/beeload-action@v1
+        with:
+          bee-url: https://api.gateway.testnet.ethswarm.org
```

Comment:
Should this be uploaded to both mainnet and testnet?

Reply:
It will probably overwrite the comment and only the one to testnet shown

Reply:
This upload does not generate the "PR Preview" comment so nothing is overridden.  The idea here is only to generate traffic on the testnet. Originally I wanted to make full switch, but there is currently no `bzz.link` on testnet and in the end we don't want to deploy it there just yet.